### ISSUE TITLE
Importing mrcz set the matplotlib backend

### ISSUE THREAD

**Issue by ericpre** [em-MRCZ/python-mrcz#8]
The `matplotlib.pyplot` import in `ReliablePy.py` set the matplotlib backend to the default setting in rcParams, which prevent setting a matplotlib backend after import mrcz.

```python
>>> import mrcz
>>> import matplotlib
>>> matplotlib.use('TkAgg')
/opt/anaconda3/lib/python3.6/site-packages/matplotlib/__init__.py:1405: UserWarning: 
This call to matplotlib.use() has no effect because the backend has already
been chosen; matplotlib.use() must be called *before* pylab, matplotlib.pyplot,
or matplotlib.backends is imported for the first time.

  warnings.warn(_use_error_msg)
```
It is possible to work around this but I guess it would be better that importing mrcz doesn't set the matplotlib backend.

**Comment by robbmcleod** [comment]
I'll make the `ReliablePy` import lazy.  I don't think anyone is using it anyway.

**Comment by robbmcleod** [comment]
Should be fixed in `0.3.6`.

**Comment by ericpre** [comment]
Thanks for the very quick fix!
